contents/tutorials/event-tracking-guide.md

```diff
@@ -3,70 +3,69 @@ title: Complete guide to event tracking
+  posthog.init('<ph_project_api_key>',{api_host:'<ph_client_api_host>',})
+</script>
+```
 
-3. Paste the block into your site’s HTML in the `<head>` tags (ideally at the end). This should be an HTML page that acts as a base or template page (with other scripts your page loads) to ensure all possible events are captured.
+> **Alternative**: Alternatively, you can install one of the SDKs such as [JavaScript Web](/docs/libraries/js), [React](/docs/libraries/react), or [React Native](/docs/libraries/react-native).
 
-This allows autocapture to capture events like clicks, change of inputs, or submission of **`a`**, **`button`**, **`form`**, **`input`**, **`select`**, **`textarea`**, and **`label`** tags. Once set up, those events flow automatically into PostHog for you to see and analyze. For a detailed guide on how to install the snippet, read our **[installation guide](/docs/integrate?tab=snippet)**.
+Once setup, this autocaptures events like clicks, change of inputs, or submission of **`a`**, **`button`**, **`form`**, **`input`**, **`select`**, **`textarea`**, and **`label`** tags. Those events flow automatically into PostHog for you to see and analyze.
 
-Autocapture can also be set up by installing the **[posthog-js](https://github.com/PostHog/posthog-js)** library, details of which can be [found here](https://posthog.com/docs/integrate/client/js), but we’ll also explain it when we cover setting up custom events below.
+> Autocapture can also capture non-event data like session duration, mouse movement, bounce rate, performance, and more. You can learn more about this in the [autocapture docs](/docs/product-analytics/autocapture).
 
 ## Autocapture’s limitations
 
-Although we are proud of our autocapture feature, we know it isn’t enough for more advanced product analytics. This is for two reasons.
+Although autocapture is a great way to get started, we realized it isn't enough for advanced event tracking. This is for three reasons:
```

Comment:
```suggestion
Although autocapture is a great way to get started, it can be limiting for more advanced event tracking. This is for three reasons:
```